app/models/outcomes.py

```diff
@@ -0,0 +1,328 @@
+import datetime
+import decimal
+
+from sqlalchemy.orm import validates, backref
+from sqlalchemy.sql.expression import (
+    and_ as sql_and,
+    case as sql_case,
+)
+
+from app import db
+from app.utils import random_positive_external_id
+from app.models import ValidationError
+
+from dmutils.formats import DATE_FORMAT, DATETIME_FORMAT
+
+
+class Outcome(db.Model):
+    __tablename__ = "outcomes"
+
+    RESULT_CHOICES = (
+        "awarded",
+        "cancelled",
+        "none-suitable",
+    )
+
+    id = db.Column(db.Integer, primary_key=True)
+    external_id = db.Column(db.BigInteger, nullable=False, default=random_positive_external_id, unique=True)
+    completed_at = db.Column(db.DateTime, nullable=True)
+
+    start_date = db.Column(db.Date, nullable=True)
+    end_date = db.Column(db.Date, nullable=True)
+    awarding_organisation_name = db.Column(db.String, nullable=True)
+    # should be enough scale to represent up to £1b - 1p
+    award_value = db.Column(db.Numeric(11, 2), nullable=True)
+
+    result = db.Column(db.String, nullable=False)
+
+    direct_award_project_id = db.Column(
+        db.Integer,
+        db.ForeignKey('direct_award_projects.id'),
+        nullable=True,
+    )
+    direct_award_search_id = db.Column(
+        db.Integer,
+        db.ForeignKey('direct_award_searches.id'),
+        nullable=True,
+    )
+    direct_award_archived_service_id = db.Column(
+        db.Integer,
+        db.ForeignKey('archived_services.id'),
+        nullable=True,
+    )
+
+    # NOTE though Outcome currently has the *ability* to be associated with a Brief/BriefResponse, at time of
+    # writing this is not yet *used* for reporting brief awards - those are still using their own mechanism, but should
+    # be ported over to this mechanism as soon as time allows.
```

Comment:
Is there a tech debt ticket for this, or is it going to be covered by mission work?

Reply:
Hopefully mission??

Reply:
(This should definitely be a tech debt ticket, but hopefully one that we should be able to address in a future mission - certainly next time we do anything outcomes/awards for briefs.)